src/platform/packages/shared/kbn-alerting-types/rule_settings.ts

```diff
@@ -35,24 +35,24 @@ export interface RulesSettingsQueryDelayProperties {
 export type RulesSettingsQueryDelay = RulesSettingsQueryDelayProperties &
   RulesSettingsModificationMetadata;
 
-export interface RulesSettingsAlertsDeletionProperties {
+export interface RulesSettingsAlertDeletionProperties {
   isActiveAlertsDeletionEnabled: boolean;
```

Comment:
should we also update the params to be AlertDeletion? I wasn't sure, could do it in another PR